examples/bzlmod/requirements.in

```diff
@@ -1,3 +1,6 @@
+--extra-index-url https://pypi.python.org/simple/
```

Comment:
is this a typo? I didn't know requirements file could have arbitrary args to pip in them.

Reply:
It was in the other example. Do you want me to remove?

Reply:
Nah, we'll just leave it be. :ghost: :headstone:

Reply:
I will note that `--extra-index-url` can be a security risk due to dependency confusion attacks. They're looking to close that vulnerability though. https://discuss.python.org/t/proposal-preventing-dependency-confusion-attacks-with-the-map-file/23414